source/computer/setup-note.rst_

```diff
@@ -0,0 +1,4 @@
+.. note::
```

Comment:
This file is only used twice, perhaps we can just put the content directly in these two files?

Reply:
Done in 579cc5f.